python/pyarrow/_azurefs.pyx

```diff
@@ -100,6 +105,9 @@ cdef class AzureFileSystem(FileSystem):
         if account_key:
             options.ConfigureAccountKeyCredential(tobytes(account_key))
             self.account_key = tobytes(account_key)
+        elif sas_token:
+            options.ConfigureSASCredential(tobytes(sas_token))
+            self.sas_token = tobytes(sas_token)
```

Comment:
Should we prefer `sas_token` to `account_key`?
(SAS token is safer than account key, right?)

Reply:
Then should we use `if sas_token: elif account_key:`?

Or should we raise an error when both of `sas_token` and `account_key` are specified?

Reply:
They should be mutually exclusive. I'll add an error later tonight.

Reply:
This is all set!

Reply:
Could you document it?